static/js/echo.js

```diff
@@ -300,6 +300,64 @@ function escape(html, encode) {
     .replace(/'/g, '&#39;');
 }
 
+/*! http://mths.be/codepointat v0.1.0 by @mathias */
```

Comment:
this third-party code should probably go either into `npm_modules` or under static/third/.  You probably should also add an entry to THIRDPARTY at the root of the repository, to properly document the origin of this third-party code.